polaris.shopify.com/src/components/HomePage/HomePage.tsx

```diff
@@ -231,6 +231,52 @@ function HomePage({}: Props) {
             </div>
           </Container>
         </div>
+        <div className={className(styles.Step, styles.PowerUps)}>
```

Comment:
We should make sure this card is still designed correctly with the updates to the home page.

Reply:
I've removed this entire section from the HomePage in lieu of the updated rebase, moved the CTA to the nav instead until we find a decent place for it, or if @martenbjork  and co have ideas around where we can put it